pegasus/helpers/section_api_helpers.rb

```diff
@@ -6,6 +6,15 @@ def self.admin?(user_id)
   end
 end
 
+class DashboardUser
+  def self.to_self_view(user_row)
```

Comment:
it looks like you were considering calling this to_self_hash... any particular rationale for calling this filtering method to_self_view and the other ones in Section to_owner_hash/etc?